tests/torch/test_pattern_manager.py

```diff
@@ -67,6 +67,7 @@
     PatternNames.LINEAR_BIASED_ELEMENTWISE: 'Not relevant for Torch.',
     PatternNames.LINEAR_ACTIVATION_ELEMENTWISE: 'Not relevant for Torch.',
     PatternNames.LINEAR_BIASED_ACTIVATION_ELEMENTWISE: 'Not relevant for Torch.',
+    PatternNames.MVN_SCALE_SHIFT_ACTIVATIONS: 'Not relevant for Torch.',
```

Comment:
Could you briefly explain why `MVN_SCALE_SHIFT_ACTIVATIONS` pattern is not relevant for Torch?

cc' @daniil-lyakhov

Reply:
Because `MVN_SCALE_SHIFT` is not relevant for Torch. We will enable these patterns in another PR